ember-headless-form/unpublished-development-types/index.d.ts

```diff
@@ -2,12 +2,23 @@
 // These will *not* be published as part of your addon, so be careful that your published code does not rely on them!
 
 import '@glint/environment-ember-loose';
+import type HeadlessFormRegistry from '../src/template-registry';
+import { ComponentLike } from '@glint/template';
```

Comment:
```suggestion
import type { ComponentLike } from '@glint/template';
```
surprised linting didn't catch this 🤔 
or maybe it doesn't matter since in a .d.ts file, everything is a type?

Reply:
Oh, I think I don't have your eslint-config for the addon (just the blueprint defaults here), only for the test-app (due to template-imports). I can add it here as well...

Reply:
I added `@nullvoxpopuli/eslint-configs` also for the addon now, and indeed `--fix` did a couple of changes!